todo-app/src/Todos.js

```diff
@@ -0,0 +1,24 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+
+export default class TodoItems extends Component {
+
+    render() {
+        const todoItems = this.props.todos;
+        
+        return (
+            <ul>
+                {todoItems.map((item, i) => {
+                    return <li key={i}>
```

Comment:
Careful about using index as a key - we discussed this in class on week 2 so you probably didn't know yet.